dagster/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app_id": "dagster-plus",
+  "app_uuid": "019635d0-d25c-7e1b-86de-83baaac55b5e",
+  "manifest_version": "2.0.0",
+  "display_on_public_website": true,
+  "tile": {
+    "title": "Dagster+",
+    "description": "Collect event logs from your Dagster+ deployments",
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Dagster+ Overview Dashboard",
+        "image_url": "images/Screenshot 2025-04-15 at 2.33.38\u202fPM.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Orchestration",
+      "Category::Log Collection",
+      "Category::Cloud",
+      "Category::AI/ML",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "oauth": {},
+  "author": {
+    "support_email": "support@dagster.cloud",
+    "name": "Dagster+",
+    "homepage": "https://dagster.io",
+    "sales_email": "sales@dagster.cloud",
+    "vendor_id": "dagster"
+  },
+  "assets": {
+    "dashboards": {
+      "Dagster+ Overview": "assets/dashboards/dagster__overview.json"
```

Comment:
nit: could you make `Dagster+ Overview` name here url safe? Whatever the string is here becomes the dashboard's "short_name" which is immutable and used in the url. Ex: https://app.datadoghq.com/dash/integration/31668/datadog-api-rate-limit-usage URL is generated where the short-name is `datadog-api-rate-limit-usage`